src/ctia/entity/investigation.clj

```diff
@@ -95,7 +95,24 @@
     :external-id-capabilities :read-investigation
     :can-aggregate?           true
     :histogram-fields         investigation-histogram-fields
-    :enumerable-fields        investigation-enumerable-fields}))
+    :enumerable-fields        investigation-enumerable-fields
+    :searchable-fields        #{:id
+                                :description
+                                :actions
+                                :external_ids
+                                :external_references.description
+                                :external_references.external_id
+                                :external_references.hashes
+                                :external_references.source_name
+                                :external_references.url
+                                :investigated_observables
+                                :object_ids
+                                :short_description
+                                :targets.observables.type
+                                :targets.observables.value
+                                :targets.os
+                                :targets.type
+                                :title}}))
```

Comment:
```suggestion
    :searchable-fields        #{:id
                                :source
                                :description
                                :investigated_observables
                                :object_ids
                                :short_description
                                :targets.observables.type
                                :targets.observables.value
                                :targets.os
                                :targets.type
                                :title}}))
```

this is an example of actions: https://github.com/threatgrid/ctia/blob/43ff86881b099bfd5a86f6a24a7ec0e05e980c04/src/ctia/entity/investigation/examples.clj#L35
```
   "actions" "[{\"uuid\":\"cae31704-9454-4428-afac-711e41a7ae24\",\"state\":\"ok\",\"updated\":\"2019-01-02T17:48:32.099Z\",\"type\":\"collect\", ...}]"})
   ```